src/riak_kv_ts_util.erl

```diff
@@ -669,17 +669,35 @@ check_table_feature_supported(DDLRecCap, DecodedReq) ->
                     {error, create_table_not_supported_message(Table)}
             end;
         {ok, _, {_, Table}} ->
-            case is_table_supported(DDLRecCap, Table) of
+            case is_table_active_and_supported(DDLRecCap, Table) of
                 true ->
                     DecodedReq;
                 false ->
-                    {error, table_not_supported_message(Table)}
+                    {error, table_not_supported_message(Table)};
+                Error ->
+                    Error
             end;
         _ ->
             DecodedReq
     end.
 
 %%
+-spec is_table_active_and_supported(DDLRecCap::atom(),
+                                    Table::binary()) ->
+                                        boolean() | {error, string()}.
+is_table_active_and_supported(DDLRecCap, Table) ->
+    Mod = riak_ql_ddl:make_module_name(Table),
+    case code:ensure_loaded(Mod) of
```

Comment:
The unit tests fail with the aforementioned output: 
```
in function ets:lookup/2
  called as lookup(metadata_manager_prefixes_ets,{core,bucket_types})
```
I'll look at those tests.  Entirely possible they don't set up a real bucket which is why they fail.